tsfresh/feature_extraction/feature_calculators.py

```diff
@@ -1204,30 +1200,29 @@ def get_kurtosis(y):
         :return type: float
         """
 
-        variance = get_variance(y)
+        var = get_variance(y)
         # In the limit of a dirac delta, kurtosis should be 3 and variance 0.  However, in the discrete limit,
         # the kurtosis blows up as variance --> 0, hence return nan when variance is smaller than a resolution of 0.5:
-        if variance < 0.5:
+        if var < 0.5:
             return np.nan
-        else:
-            return (
-                get_moment(y, 4)
-                - 4 * get_centroid(y) * get_moment(y, 3)
-                + 6 * get_moment(y, 2) * get_centroid(y) ** 2
-                - 3 * get_centroid(y)
-            ) / get_variance(y) ** 2
-
-    calculation = dict(
-        centroid=get_centroid,
-        variance=get_variance,
-        skew=get_skew,
-        kurtosis=get_kurtosis,
-    )
+        return (
```

Comment:
As you are touching it anyways ;)
Can you extract a variable for the y centroid? It only needs to be calculated once

Reply:
https://github.com/blue-yonder/tsfresh/pull/1103/commits/5b26355456658fad9444fe773c51b9edf55ae2de